apps/playground-web/src/app/api/airdrop/route.ts

```diff
@@ -0,0 +1,155 @@
+import { Engine } from "@thirdweb-dev/engine";
+import * as dotenv from "dotenv";
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+dotenv.config();
+
+const CHAIN_ID = "84532";
+const BACKEND_WALLET_ADDRESS = process.env.ENGINE_BACKEND_WALLET as string;
+
+console.log("Environment Variables:");
```

Comment:
Remove logs

Reply:
Removed this file entirely, it is unused, removed logs from the rest as well